### ISSUE TITLE
Star Wars: Jedi Starfighter - (Mostly) silent end-of-mission cutscenes

### ISSUE THREAD

**Issue by MageKing17** [PCSX2/pcsx2#1928]
***PCSX2 version:***
`pcsx2-v1.5.0-dev-2055-g29eed18-windows-x86`

***PCSX2 options:***
As far as I can tell, no options affect the issue.

***Plugins used:***
`SPU2-X 20170430234119 2.0.0 [Spu2-X]`
The timeout issue occurs even in "No Sound" mode.

***Description of the issue:***
Most missions (not training missions, bonus missions, or ones that end in an FMV) have an in-engine cutscene after completion, which contains some dialogue; for some reason, the audio for these cutscenes are glitched and no voices or sound effects play (although, for some reason, you can still hear the sounds of ships' engines... possibly because they're looping sounds?), and the game will seemingly freeze for 15 seconds before the cutscene plays, with PCSX2 logging some errors in the process.

Here is what the log window shows at the end of the first mission: https://pastebin.com/DuAHJJWK

(The `padman: *** VBLANK OVERLAP ***` line doesn't appear to be unique to this situation, but it does get repeatedly logged during the 15 second wait, hence why I included it.)

***How to reproduce the issue:***
1. Complete any campaign mission that doesn't transition straight into an FMV.
2. Wait at the black screen for 15 seconds.
3. Watch the mostly-silent cutscene.

***Last known version to work:***
Problem occurs at least as far back as 1.2.1, which is the oldest version I currently have downloaded.

***PC specifications:***
CPU: `AMD 9590-FX` (underclocked to ~4.2 GHz/core, if that matters)
GPU: `R9 290`
OS: `Windows 10 64-bit`

**Comment by ramapcsx2** [comment]
http://forums.pcsx2.net/Thread-Audio-Error-In-Cutscenes

Quite a verbose game. Lots of invalid NAX and SSA logs and even an IOP mem dump!

**Comment by atomic83GitHub** [comment]
Did you test this PR? https://github.com/PCSX2/pcsx2/pull/2275

**Comment by MageKing17** [comment]
@atomic83GitHub So I somehow missed the notification email back when you asked, but I just tried out the PR you mentioned and it doesn't affect this issue it all.

**Comment by MrCK1** [comment]
@MageKing17 Can you retest both your issues on the latest dev build and see if they still occur? Thanks.

**Comment by MageKing17** [comment]
Just fired up `PCSX2 v1.7.0-dev-767-g2c7f54134` and both this and #1927 are still present.

**Comment by prafullpcsx2** [comment]
Can you retest latest dev build with this? There have been some spu2 fixes which may fix the problem. Also if possible post a savestate right before the problem using latest dev build.

**Comment by MageKing17** [comment]
Issue still present in `PCSX2 v1.7.0-dev-817-gc998a51f7`; attached is a save state at completion of the first mission, immediately before the ending cutscene:
[Issue1928.zip](https://github.com/PCSX2/pcsx2/files/5756552/Issue1928.zip)
The log contents are largely the same as the pastebin from the original post, except that the IOP memory dump is different.

**Comment by refractionpcsx2** [comment]
Can this be tried again? There was more SPU fixes after this was last tested.

**Comment by MageKing17** [comment]
Just ran `PCSX2 Nightly - v1.7.2731 Compiled on May  9 2022`, and the issue is unchanged.

**Comment by Daedliy** [comment]
> Just ran `PCSX2 Nightly - v1.7.2731 Compiled on May 9 2022`, and the issue is unchanged.

v1.7.3043 and it's still an issue unfortunately

**Comment by SlyCooperReloadCoded** [comment]
A long time ago in the 1.4.0 days I remember seeing the solution posted somewhere on the forums and I've been using it ever since to fix this issue - increasing EE Cyclerate to 130%.  Just tested it again on 1.7.3220 and yeah, it still works.  Granted it's probably just a workaround and not addressing the root cause, but it works.

**Comment by MageKing17** [comment]
Just tried this in v1.7.4280 and, for the first time in PCSX2, heard dialogue in an end-of-mission cutscene. It would seem that, at some point in the past few months, this issue got fixed.

**Comment by refractionpcsx2** [comment]
I think it did work sometimes, can you repeat completing the mission?

**Comment by MageKing17** [comment]
No, this issue wasn't one that only occurred at random; you may be thinking of #1927 (which is unfortunately still present).

**Comment by refractionpcsx2** [comment]
oh, maybe you're right, sorry!

**Comment by MageKing17** [comment]
Apparently I stand corrected about it happening at random, because when I went back to do a binary search to find which build fixed the issue, it didn't work even on the "known-good" build. I can confirm the 130% EE overclock seems to work reliably, though.

**Comment by MrCK1** [comment]
Can this be retested?

**Comment by SlyCooperReloadCoded** [comment]
Re-tested on latest nightly, no longer an issue.

**Comment by foxsevent** [comment]
As reported by Flowden on discord.
https://discord.com/channels/309643527816609793/923492653788692480/1446214938015043795

This issue still persist in 2.4.0 and 2.5.349, seems that only setting the EE cycle rate to anything other than 100% or disabling EE recompiler fixes it.
https://youtu.be/YkH_DpBOdhY

The issue seems always to happen on 2.4.0 and newer versions but on older version the results were inconsistent, sometimes it happen and sometimes it didn't making testing difficult.
One thing I can add is that the issue can be baked into a save state before the cut scene loads but the state can be fixed by the above workaround as long as the state is before the cut scene load.

In this situation the ticket should be re-opened I think.

**Comment by SlyCooperReloadCoded** [comment]
This is very odd. I went and re-tested this on latest (v2.5.350) with stock settings, 100% EE Cycle Rate included, and this issue does not occur for me.  Cutscenes work like normal.

**Comment by foxsevent** [comment]
> This is very odd. I went and re-tested this on latest (v2.5.350) with stock settings, 100% EE Cycle Rate included, and this issue does not occur for me. Cutscenes work like normal.

Well from testing other versions I noted the issue being inconsistent, so I may have been just very unlucky with the newer versions.
Just tested 2.5.350 and same issue.
This was on the very first mission and my copy SLUS-20293, maybe there's a difference with other missions or versions of the game.
